src/prop-converter.js

```diff
@@ -28,18 +33,41 @@ import {
   disablePreset,
 } from './enter-leave-presets';
 import { isElementAnSFC, omit } from './helpers';
+import type {
+  Animation,
+  AnimationProp,
+  Presets,
+  FlipMoveProps,
+  FlipMoveDefaultProps,
+  ConvertedProps,
+  DelegatedProps,
+} from './typings';
+
+const nodeEnv: string = process && process.env && process.env.NODE_ENV
+  ? process.env.NODE_ENV
+  : 'development';
+
+function propConverter(
+  ComposedComponent: Class<Component<*, ConvertedProps, *>>
+): Class<Component<FlipMoveDefaultProps, FlipMoveProps, void>> {
+  return class FlipMovePropConverter extends Component {
+    static defaultProps = {
+      easing: 'ease-in-out',
+      duration: 350,
+      delay: 0,
+      staggerDurationBy: 0,
+      staggerDelayBy: 0,
+      typeName: 'div',
+      enterAnimation: defaultPreset,
+      leaveAnimation: defaultPreset,
+      disableAllAnimations: false,
+      getPosition: node => node.getBoundingClientRect(),
+      maintainContainerHeight: false,
+      verticalAlignment: 'top',
+    };
 
-
-let nodeEnv;
-try {
-  nodeEnv = process.env.NODE_ENV;
-} catch (e) {
-  nodeEnv = 'development';
-}
-
-function propConverter(ComposedComponent) {
-  class FlipMovePropConverter extends Component {
-    checkForStatelessFunctionalComponents(children) {
+    // eslint-disable-next-line class-methods-use-this
+    checkForStatelessFunctionalComponents(children: mixed) {
```

Comment:
Exactly. `mixed` means that the value is opaque and you can make no assumptions about it